index.js

```diff
@@ -68,6 +73,13 @@ function getEarthRadius(options) {
   return defaultEarthRadius;
 }
 
+function getDirection(options){
+  if (isObjectNotArray(options) && options.rightHandRule === true){
```

Comment:
you don't need the `=== true`

Reply:
`true`. I did it only to distinguish from other truth values, but honestly, then they have bigger problems than the circle going in the wrong order.

Fixed here: https://github.com/gabzim/circle-to-polygon/pull/37/commits/9e36f69d4a4f6a8621fe5d397b188e49f52a7c24